starboard/tools/abstract_launcher.py

```diff
@@ -95,8 +96,42 @@ def LauncherFactory(platform_name,
       **kwargs)
 
 
+class ReturnCodeStatus(IntEnum):
+  """Represents status of code, returned by target"""
```

Comment:
Conceptually this lgtm - but should we somehow modify or amend `def Run` signature to indicate it's expected to return one of these values ? Or at least add a comment in the function doc.

Reply:
Extended the comment in the function Run. See commit [5274e5c55f6](https://github.com/youtube/cobalt/commit/5274e5c55f638d43e06433e4e66b4b8428e89f3a). I left return value of Run as-is to avoid possible issues in dependent implementations. I consider deprecation of `def Run`.